rsmtool/input.py

```diff
@@ -1410,7 +1411,8 @@ def load_and_filter_data(input_file,
                                                      exclude_zero_sd=exclude_zero_sd)
             del df_filtered
             df_filtered = newdf
-            df_excluded = pd.merge(df_excluded, newdf_excluded, how='outer')
+            with np.errstate(divide='ignore'):
```

Comment:
This happens inside `pd.merge` when you are merging an empty data from with an empty data frame.